fetch/_core.py

```diff
@@ -608,7 +628,7 @@ def process(self, file_path):
             raise FileProcessError('Return code %r from command %r' % (returned, command))
 
         # Check that output exists
-        expected_path = self._apply_file_pattern(self.expect_file, file_path)
+        expected_path = self._apply_file_pattern(self.expect_file, file_path, **required_files_formating)
```

Comment:
This is the line that required a change, but looks like you've already done it :)